src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -67,11 +68,19 @@ final class ValidatorPropertyMetadataFactory implements PropertyMetadataFactoryI
 
     private $decorated;
     private $validatorMetadataFactory;
+    /**
+     * @var iterable<PropertySchemaRestrictionMetadataInterface>
+     */
+    private $restrictionsMetadata;
 
-    public function __construct(ValidatorMetadataFactoryInterface $validatorMetadataFactory, PropertyMetadataFactoryInterface $decorated)
+    /**
+     * @param PropertySchemaRestrictionMetadataInterface[] $restrictionsMetadata
+     */
+    public function __construct(ValidatorMetadataFactoryInterface $validatorMetadataFactory, PropertyMetadataFactoryInterface $decorated, iterable $restrictionsMetadata)
```

Comment:
Can we default iterable $restrictionsMetadata to an array? I fear it may break some implementations if we leave this as-is.

Reply:
@soyuka I've added default value